specification/managementgroups/resource-manager/Microsoft.Management/preview/2023-12-01-alpha/management.json

```diff
@@ -0,0 +1,252 @@
+{
+  "swagger": "2.0",
+  "host": "management.azure.com",
+  "info": {
+    "version": "2023-12-01-alpha",
+    "title": "Management Groups APIs for Applications",
+    "description": "The Azure Management Groups API enables consolidation of multiple \nsubscriptions/resources into an organizational hierarchy and centrally \nmanage access control, policies, alerting and reporting for those resources.\n This API version includes the preview state of Applications which are a new version of Management Groups.\n"
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "tags": [
+    {
+      "name": "Applications",
+      "description": "An Application is a customer created scope, where multiple subscriptions or applications can be grouped by. \n"
+    }
+  ],
+  "paths": {
```

Comment:
please fix all lint errors in https://github.com/Azure/azure-rest-api-specs/pull/27033/checks?check_run_id=19524864649